packages/client/src/test.cairo

```diff
@@ -56,11 +53,35 @@ fn test(mut arguments: Span<felt252>) {
 
     if state.chain_state != expected_chain_state {
         println!(
-            "FAIL: block={} error='expected state {:?}, actual {:?}'",
+            "FAIL: block={} error='expected chain state {:?}, actual {:?}'",
             state.chain_state.block_height,
             expected_chain_state,
             state.chain_state
         );
         panic!();
     }
+
+    gas_before = get_available_gas();
+
+    match state.utreexo_state.validate_and_apply(utxo_set) {
```

Comment:
Let's disable this for now, only utxo set checks (double spending / empty cache) have to be triggered.